src/parser/ast/FunctionNode.h

```diff
@@ -84,6 +88,28 @@ class FunctionNode : public StatementNode {
         }
     }
 
+    void generateFunctionNameByteCode(ByteCodeBlock* block, ByteCodeGenerateContext* context)
+    {
+        InterpretedCodeBlock* codeBlock = block->codeBlock();
+        AtomicString name = codeBlock->functionName();
+        if (name.string()->length()) {
+            if (UNLIKELY(codeBlock->isFunctionNameExplicitlyDeclared())) {
```

Comment:
Should we use `UNLIKELY` in both cases?
```
            if (UNLIKELY(codeBlock->isFunctionNameExplicitlyDeclared())) {
...
            } else if (UNLIKELY(codeBlock->isFunctionNameSaveOnHeap() && !name.string()->equals("arguments"))) {
```

Reply:
IMO both cases are rarely occurred so `UNLIKELY` is used in both statements.
(to be specific, function name value is usually allocated in the caller scope, but for some cases, it needs to be allocated in the callee scope and this part is for that case)